### PR TITLE
noir 0.12.0

### PR DESCRIPTION
**Pull request by hahwul**
Created with `brew bump-formula-pr`.<details>
  <summary>release notes</summary>
  <pre>## What's Changed
### New Features
- FileAnalyzer and Hooks: 
  - Introducing a powerful addition to enhance detection capabilities through comprehensive file content analysis.
  - The FileAnalyzer operates independently of the detected language or framework, enabling file-based scans. This feature utilizes Hooks to perform advanced actions such as detecting URLs within strings, Base64 decoding, and more. These Hooks contribute to additional endpoint identification, augmenting the overall analysis.
- go-fiber detector & analyzer: Added support for the [go fiber](https://gofiber.io) web framework, expanding compatibility.

### Enhancements
- `--include-path` Flag: This flag also reveals the original source code file of detected endpoints when used.
- In JSON/YAML format, the Details section allows you to explore the original source code file of detected endpoints.

### Analyzer Improvements
- JS Express Analyzer: Enhanced for more accurate and insightful results.
- Spring (Java/Kotlin) Analyzer: Improved precision for a better understanding of Spring framework projects.

### Removal
- `--scope` Flag: Deprecated (It was a flag that actually meant nothing.)

### Code Quality
- Code Improvements: Refined codebase for better quality. and more test codes

### User Experience
- CLI Banner Update: Refreshed for a more appealing appearance.

**Full Changelog**: https://github.com/noir-cr/noir/compare/v0.11.0...v0.12.0</pre>
</details>

### CODE REVIEW

File: Formula/n/noir.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Noir < Formula
   desc "Attack surface detector that identifies endpoints by static analysis"
   homepage "https://github.com/noir-cr/noir"
-  url "https://github.com/noir-cr/noir/archive/refs/tags/v0.11.0.tar.gz"
-  sha256 "37b0b78d4673cc2482ad346010de8b36be5f75c92724938aa6e2ec1fd1883e20"
+  url "https://github.com/noir-cr/noir/archive/refs/tags/v0.12.0.tar.gz"
+  sha256 "25660741e04a7c4e617fcb1fbfdbb85568665b13126325bde58545dffda7a935"
   license "MIT"
   revision 1
```
Comment (bevanjkay) [inline Formula/n/noir.rb:7]:
```suggestion
```

Revision needs to be removed

**Review by bevanjkay — CHANGES_REQUESTED** [review]

**Comment by hahwul** [comment]
Hi @bevanjkay 
I'm having trouble squash the commits, and I think it's because of the merge commit. Can you give me some instructions on how to solve this problem?

**Comment by bevanjkay** [comment]
Please don't ping all of the teams and project leader for a review. Maintainers will review your PR in due time. 

You need to rebase and squash your commits instead of creating merge commits.

**Comment by hahwul** [comment]
@bevanjkay 
I'm sorry. I didn't mean to ping. It seems like I got caught up in my work while using GitHub Desktop App.
I performed a rebase and squashed the commits as you advised. Thanks a lot for your guidance!

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**